### ISSUE TITLE
Bugs and dupe

### ISSUE THREAD

**Issue by PixelmonPRO** [AlmuraDev/SGCraft#36]
Hello! Tested SGCraft in server (and singleplayer), and make video in errors / bugs and dupe.
https://www.youtube.com/watch?v=465dq4uTwYU&feature=youtu.be
Version SG: 1.15.5 Forge (play in Forge 2705)
And Server: SG 1.15.5 Forge / SF 3206

**Comment by Dockter** [comment]
Can you please write out what exactly it is you're seeing?  With no video dialog its very hard to follow.

**Comment by TherayTharow** [comment]
I did a small test my self based on what i saw done 
you have a chest minecart that you open just before it goes though
so when it does remove the stuff from the "chest" then go to the other side and open the chest with all the same stuff inside so now you have 2 what was in the chest at tbe start

**Comment by TherayTharow** [comment]
same usest to happen with donkeys and end portels in 1.8.1 i think
this looks like the same type of problem

**Comment by TherayTharow** [comment]
@dockter https://minecraft.gamepedia.com/Tutorials/Block_and_item_duplication. 
under Java Edition
Deadly Method

**Comment by PixelmonPRO** [comment]
@Dockter I can reproduce items duplication, after update my server.
Version SG: 1SGCraft-1.15.6-forge-release (play in Forge 2768)
And Server: SGCraft-1.15.6-forge-release / spongeforge-1.12.2-2768-7.1.5

**Comment by Dockter** [comment]
Ok, the dup issue is fixed as of 1.20.0.  As for the rest of this, since it wasn't actually typed out as to what the issue is and I can't amek heads or tails of what is being shown, I'm closing this.
